### PR TITLE
Fix contributing rst

### PR DESCRIPTION
**Pull request by tbetbetbe**
To be submitted after #5

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -1,13 +1,19 @@
 Contributing
 ============
 
-1.  **Please sign one of the contributor license agreements [below][6].**
-1.  [File an issue][9] to notify the maintainers about what you're working on.
-1.  [Fork the repo][10], develop and [test][11] your code changes, add docs.
-1.  Make sure that your commit messages clearly describe the changes.
-1.  [Send][12] a pull request.
+Here are some guidelines for hacking on `endpoints-management-python`_.
```
Comment (lookuptable) [inline CONTRIBUTING.rst]: Consider changing "hacking on" to "contributing to"

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
+-  `File an issue`_ to notify the maintainers about what you're working on.
+-  `Fork the repo`_; develop and `test your code changes`_; add docs.
+-  Make sure that your `commit messages`_ clearly describe the changes.
+-  `Make the pull request`_.
+
+.. _`Fork the repo`: https://help.github.com/articles/fork-a-repo
+.. _`forking`: https://help.github.com/articles/fork-a-repo
+.. _`commit messages`: http://chris.beams.io/posts/git-commit/
+
+.. _`File an issue`:
```
Comment (lookuptable) [inline CONTRIBUTING.rst]: Link is missing

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -43,83 +54,72 @@ Using a Development Checkout
...
+    git clone git@github.com:USERNAME/endpoints-management-python.git hack-on-endpoints-management-python
+    cd hack-on-endpoints-management-python
+
+    # Configure remotes such that you can pull changes from the endpoints-management-python
+    # repository into your local repository.
+    git remote add upstream https://github.com:google/endpoints-management-python
+
+    # fetch and merge changes from upstream into master
+    git fetch upstream
+    git merge upstream/master
```
Comment (lookuptable) [inline CONTRIBUTING.rst]: I'm wondering whether we can remove all the details and just refer readers to https://guides.github.com/introduction/flow/

File: CONTRIBUTING.rst (hunk 1 above)
Comment (tbetbetbe) [inline CONTRIBUTING.rst]: Done

File: CONTRIBUTING.rst (hunk 2 above)
Comment (tbetbetbe) [inline CONTRIBUTING.rst]: That's intended.  This makes it an internal target that is linked to from some referencing text above.

**Comment by lookuptable** [comment]
👍

**Pull request merged**